the-events-calendar.php

```diff
@@ -27,11 +27,10 @@
 Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
 
+define( 'THE_EVENTS_CALENDAR_DIR', dirname( __FILE__ ) );
+define( 'THE_EVENTS_CALENDAR_FILE', __FILE__ );
```

Comment:
Interesting idea. Would you include the abbreviation? Eitherway it is probably unique enough to not conflict in the global namespace. 

Many of our plugins already have the constant present in the same format as I used.